docs/docusaurus/docs/guides/setup/configuring_metadata_stores/how_to_configure_a_metricsstore.md

```diff
@@ -106,19 +107,13 @@ action_list:
         - statistics.unsuccessful_expectations
 ```
 
-### 3. Test your MetricStore and StoreMetricsAction
+## 3. Test your MetricStore and StoreMetricsAction
 
-To test your `StoreMetricsAction`, run your Checkpoint from your code:
+Run the following command to run your Checkpoint and test `StoreMetricsAction`:
 
 ```python
```

Comment:
Another code block to flag as needing to be pulled from an example script under test.

Reply:
@Rachel-Reverie a note has been added for this code block.